src/__tests__/license.test.js

```diff
@@ -0,0 +1,34 @@
+const { License } = require('../components/license.js')
+
+test('MIT license badge title displays correctly', () => {
+  var license = new License('MIT')
```

Comment:
Here I made License its own object.

This is actually not that important to me - I had the app working fine as is for license.